web_responsive/static/src/js/web_responsive.js

```diff
@@ -264,6 +265,14 @@ odoo.define('web_responsive', function (require) {
                 },
             });
         },
+
+        // Load view indicator
+        _onAppsMenuItemClicked: function (ev) {
+            ev.preventDefault();
+            ev.stopPropagation();
+            $(ev.currentTarget).find('img').addClass('o-app-icon-waiting');
```

Comment:
I'd change it like this:

```suggestion
            $(ev.currentTarget).addClass('o-app-icon-waiting');
```

Then the CSS above can just apply to `.o-app-icon-waiting img`. It gives you a little bit more flexibility in case you want i.e. add more styling to the whole block.